src/panels/logbook/ha-panel-logbook.ts

```diff
@@ -17,6 +17,7 @@ import "../../components/ha-date-range-picker";
 import type { DateRangePickerRanges } from "../../components/ha-date-range-picker";
 import "../../components/ha-icon-button";
 import "../../components/ha-menu-button";
+import { loadAutomationTraceContexts } from "../../data/automation_debug";
```

Comment:
```suggestion
import { AutomationTraceContexts, loadAutomationTraceContexts } from "../../data/automation_debug";
```

This will probably make the linter unhappy